migration/src/spreadsheets.rs

```diff
@@ -176,7 +178,9 @@ impl SheetResult {
                     None
                 };
 
-                let intro_or_body = if is_intro { Intro } else { Body };
+                let intro_or_body = if chapter_type==0 { Intro } 
```

Comment:
Slight nit, but would just space it out so it looks like `chapter_type == 0` for readability!

Reply:
And I would maybe rename the variable `intro_or_body` to `chapter_type_name` since it can also be a `Credit` and the variable name could be misleading

Reply:
good catch, thanks!!